### PR TITLE
fix: use info icon for overwrite prevented status instead of success icon

### PR DESCRIPTION
**Pull request by jordanvn**
#### Description of changes
* replaces use of "success" icon with "info" icon for `OVERWRITE_PREVENTED` status

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-storage/src/components/StorageBrowser/views/LocationActionView/getActionIcon.ts [hunk 1]
```diff
@@ -6,13 +6,15 @@ export const getActionIcon = (status: TaskStatus | 'CANCELED'): IconVariant => {
...
-    case 'COMPLETE':
-    case 'OVERWRITE_PREVENTED': {
+    case 'COMPLETE': {
       return 'action-success';
     }
     case 'FAILED': {
       return 'action-error';
     }
+    case 'OVERWRITE_PREVENTED': {
+      return 'info';
```
Comment (calebpollman) [inline packages/react-storage/src/components/StorageBrowser/views/LocationActionView/getActionIcon.ts:16]: So these icons names stay aligned let's add an 'action-info` icon variant that uses the underlying 'info' icon

File: packages/react-storage/src/components/StorageBrowser/context/elements/IconElement.tsx [hunk 2]
```diff
@@ -44,6 +45,9 @@ export const DEFAULT_ICON_PATHS: Record<IconVariant, string> = {
   // "Error"
   'action-error':
     'M480-280q17 0 28.5-11.5T520-320q0-17-11.5-28.5T480-360q-17 0-28.5 11.5T440-320q0 17 11.5 28.5T480-280Zm-40-160h80v-240h-80v240Zm40 360q-83 0-156-31.5T197-197q-54-54-85.5-127T80-480q0-83 31.5-156T197-763q54-54 127-85.5T480-880q83 0 156 31.5T763-763q54 54 85.5 127T880-480q0 83-31.5 156T763-197q-54 54-127 85.5T480-80Zm0-80q134 0 227-93t93-227q0-134-93-227t-227-93q-134 0-227 93t-93 227q0 134 93 227t227 93Zm0-320Z',
+  // "Info"
+  'action-info':
+    'M440-280h80v-240h-80v240Zm40-320q17 0 28.5-11.5T520-640q0-17-11.5-28.5T480-680q-17 0-28.5 11.5T440-640q0 17 11.5 28.5T480-600Zm0 520q-83 0-156-31.5T197-197q-54-54-85.5-127T80-480q0-83 31.5-156T197-763q54-54 127-85.5T480-880q83 0 156 31.5T763-763q54 54 85.5 127T880-480q0 83-31.5 156T763-197q-54 54-127 85.5T480-80Zm0-80q134 0 227-93t93-227q0-134-93-227t-227-93q-134 0-227 93t-93 227q0 134 93 227t227 93Zm0-320Z',
```
Comment (jordanvn) [inline packages/react-storage/src/components/StorageBrowser/context/elements/IconElement.tsx:50]: This is the same as the existing `info`

File: packages/react-storage/src/components/StorageBrowser/views/LocationActionView/getActionIcon.ts (hunk 1 above)
Comment (jordanvn) [reply packages/react-storage/src/components/StorageBrowser/views/LocationActionView/getActionIcon.ts]: Updated

File: packages/react-storage/src/components/StorageBrowser/context/elements/IconElement.tsx (hunk 2 above)
Comment (calebpollman) [reply packages/react-storage/src/components/StorageBrowser/context/elements/IconElement.tsx:50]: Nit: could just assign the path value to a single constant and use for both 'info' and 'action-info'

**Review by calebpollman — APPROVED** [review]

**Pull request merged**